cnxauthoring/views.py

```diff
@@ -106,12 +110,14 @@ def logout(request):
     raise httpexceptions.HTTPFound(location=redirect_to)
 
 
-@view_config(route_name='options', request_method='OPTIONS', renderer='string')
+@view_config(route_name='options', request_method='OPTIONS',
+             renderer='string', http_cache=DEFAULT_CACHE)
 def options(request):
```

Comment:
No, this needs to be available at any url and whether you're authenticated or not.  Looking at the commit message "Respond to OPTIONS with CORS headers" it was about letting webview use authoring api.